libs/domain/cart/totals/src/totals.schema.ts

```diff
@@ -7,4 +8,9 @@ export const cartTotalsComponentSchema: ContentComponentSchema<CartTotalsCompone
     name: 'Cart totals',
     group: 'Cart',
     icon: IconTypes.BulletList,
+    options: {
+      reference: {
+        type: FormFieldType.Text,
```

Comment:
What about making this a select with cart and order option?

Reply:
Check next comment, please